vendor/processors/stripe_processor.py

```diff
@@ -199,6 +269,203 @@ def build_subscription(self, subscription, payment_method_id):
             'metadata': {'site': self.invoice.site}
         }
 
+
+    ##########
+    # Stripe & Vendor Objects Synchronization
+    ##########
+
+    def get_stripe_customers(self):
+        """
+        Returns all Stripe created customers
+        """
+        query = self.build_search_query([{
+            'key_name': 'site',
+            'key_value': self.site.domain,
+            'field_type': 'metadata'
+        }])
+        customer_search = self.stripe_query_object(self.stripe.Customer, {'query': query})
+
+        if customer_search:
+            return customer_search['data']
+
+        return None
+
+    def get_stripe_customers_in_vendor(self, customers):
+        """
+        Returns all vendor customers who have been created as Stripe customers
+        """
+        email_list = [customer_obj['email'] for customer_obj in customers]
+        stripe_users = CustomerProfile.objects.filter(
+            user__email__iregex=r'(' + '|'.join(email_list) + ')',  # iregex used for case insensitive list match
+            site=self.site
+        )
+
+        return stripe_users
+
+    def get_stripe_customers_not_in_vendor(self, customers):
+        """
+        Returns all vendor customers who have not been created as Stripe customers
+        """
+        email_list = [customer_obj['email'] for customer_obj in customers]
+        stripe_users = CustomerProfile.objects.exclude(
+            user__email__iregex=r'(' + '|'.join(email_list) + ')',  # iregex used for case insensitive list match
+            site=self.site
+        )
+
+        return stripe_users
+
+    def create_stripe_customer_not_in_vendor(self, customers):
```

Comment:
Rename this function to `create_stripe_customers` and you can remove the comment